salt/utils/pkg/win.py

```diff
@@ -75,9 +75,14 @@
 try:
     from salt.utils.versions import LooseVersion
 except ImportError:
-    from setuptools._distutils.version import (
-        LooseVersion,  # pylint: disable=blacklisted-module
-    )
+    try:
+        from setuptools.distutils.version import (
+            LooseVersion,  # pylint: disable=blacklisted-module
+        )
+    except ImportError:
+        from setuptools._distutils.version import (
+            LooseVersion,  # pylint: disable=blacklisted-module
+        )
```

Comment:
```suggestion
from salt.utils.versions import LooseVersion
```

Reply:
Just updated and submitted, can you look at recent code

Reply:
done